README.md

```diff
@@ -0,0 +1,11 @@
+__**CascadeBot**__
+
+CascadeBot runs on JDK 11, which is good because yeet? Apparently I also need to mention the rage that Ryan has thrown at it, not sure who Ryan is though :D
```

Comment:
Runs on jdk 11 because jdk 10 is no longer supported and we needed jdk 10 features so only option was jdk 11